### PR TITLE
add iot team as codeowners for iot sdk

### PR DESCRIPTION
**Pull request by danewalton**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -21,6 +21,7 @@
 /sdk/platform/http_client/  @vhvb1989
 /sdk/platform/win32/        @antkmsft @jeffreyrichter
 /sdk/storage/               @gilbertw
+/sdk/iot/                   @ericwol-msft @jspaith @ewertons @hihigupt @CIPop @danewalton
```
Comment (antkmsft) [inline .github/CODEOWNERS]:
```suggestion
/sdk/iot/                   @CIPop @danewalton @ericwol-msft @ewertons @hihigupt @jspaith
```

Comment (antkmsft) [reply .github/CODEOWNERS]: But up to you, this is really minor thing of no technical difference.

Comment (danewalton) [reply .github/CODEOWNERS]: good rec. updated

Comment (antkmsft) [reply .github/CODEOWNERS]: Thank you Dane! I'll complete your PR soon as CI/CD finish rebuilding.

Comment (danewalton) [reply .github/CODEOWNERS]: just to remove an interpretation of hierarchy in the listed names. good suggestion.

**Pull request merged**